core/src/main/scala/com/github/gvolpe/fs2redis/interpreter/Fs2Redis.scala

```diff
@@ -174,7 +174,14 @@ private[fs2redis] class Fs2Redis[F[_], K, V](val client: StatefulRedisConnection
   override def mGet(keys: Set[K]): F[Map[K, V]] =
     JRFuture {
       F.delay(client.async().mget(keys.toSeq: _*))
-    }.map(_.asScala.toList.map(kv => kv.getKey -> kv.getValue).toMap)
+    }.map {
+      _.asScala.toList.flatMap { kv =>
```

Comment:
I think we can still keep the implementation in a single line defined like this:

```scala
JRFuture {
  F.delay(client.async().mget(keys.toSeq: _*))
}.map(_.asScala.toList.map { case kv if kv.hasValue => kv.getKey -> kv.getValue }.toMap)
```

Reply:
used `collect ` as `map` will result in MatchError